app/assets/translations/appConstants/bebbo/KGru.ts

```diff
@@ -456,7 +457,7 @@ const KGru = {
                 tncScreenContent: "Беббо не собирает и не хранит личную информацию. Все введенные вами данные  сохраняются только на вашем телефоне.",
                 welcomeText:"Добро пожаловать!",
                 letGetStartedText:"ДАВАЙТЕ НАЧНЕМ!",
-                searchInBebboText : "Search in Bebbo",
+                searchInBebboText : "Search",
```

Comment:
You should probably apply the same change in KGky.ts file.

Reply:
Please ignore 'search' string for now as we are not using it.